worker/tasks.py

```diff
@@ -0,0 +1,524 @@
+#
+# This is a tasks.py file for the pip invoke package: https://docs.pyinvoke.org/.
+#
+# It's a replacement of our Makefile with same tasks.
+#
+# Usage:
+#   invoke --list
+#
+
+import sys;
+import os;
+import inspect;
+import shutil;
+import subprocess;
+import re;
+from invoke import task;
+
+
+MEDIASOUP_BUILDTYPE = os.getenv('MEDIASOUP_BUILDTYPE') or 'Release';
+WORKER_DIR = os.path.dirname(os.path.abspath(
+    inspect.getframeinfo(inspect.currentframe()).filename
+));
+# NOTE: MEDIASOUP_OUT_DIR is overrided by build.rs.
+MEDIASOUP_OUT_DIR = os.getenv('MEDIASOUP_OUT_DIR') or f'{WORKER_DIR}/out';
+MEDIASOUP_INSTALL_DIR = os.getenv('MEDIASOUP_INSTALL_DIR') or f'{MEDIASOUP_OUT_DIR}/{MEDIASOUP_BUILDTYPE}';
+BUILD_DIR = os.getenv('BUILD_DIR') or f'{MEDIASOUP_INSTALL_DIR}/build';
+PIP_DIR = f'{MEDIASOUP_OUT_DIR}/pip';
+NUM_CORES = re.findall(
+    r'\b\d+\b', subprocess.check_output(f'{WORKER_DIR}/scripts/cpu_cores.sh', text=True)
```

Comment:
This is failing in Windows probably because it trying to execute the executable cpu_cores.sh directly. I could try to fix it by adding sh at the beginning but I have no idea if that's the way to run a bash script in Windows. Any tip?

Reply:
In the Makefile we call that .sh script with "shell" but now we are not in Makefile anymore and I don't know how make for Windows translates that "shell" (and we don't have make for Windows anymore).

Reply:
Ignore, I can use `len(os.sched_getaffinity(0))` (if available) or `os.cpu_count()`.

Reply:
Yes, let's remove shell scripts if we can use a nicer language to get the same data

Reply:
Done.